src/test/java/com/list/todo/services/TodoListServiceTest.java

```diff
@@ -182,33 +245,40 @@ public void changeTodoListStatus_OnExistentTodoList_TodoList() {
         todoListService.changeTodoListStatus(TODO_LIST_ID, TodoListStatus.INACTIVE);
 
         // assert
-        verify(todoList).setTodoListStatus(TodoListStatus.INACTIVE);
-        verify(todoListRepository).save(todoList);
+        verify(todoList, times(1)).setTodoListStatus(TodoListStatus.INACTIVE);
+        verify(todoListRepository, times(1)).save(todoList);
     }
 
     @Test
-    public void deleteTodoList_Successful() {
+    public void deleteTodoList_DeleteNotSharedTodoList_Void() {
```

Comment:
it is not mandatory to follow string pattern when naming tests. I mean, this `_Void` part does not give any valuable information.
Yes, you have to specify result of the method invocation in test name, but RESULT does not necessarily mean RETURN_VALUE.
In this particular case result is **deletion of not shared todoList**
So you can name test `deleteTodoList_DeleteNotSharedTodoList_deletesTodoList`, or `deleteTodoList_DeleteNotSharedTodoList_happyPath`, or just `deleteTodoList_DeleteNotSharedTodoList`